enterprise/app/settings/settings.css

```diff
@@ -144,12 +154,24 @@
 }
 
 .settings .settings-tab {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
   padding: 8px 16px;
   position: relative;
   overflow: hidden;
   transition: box-shadow 100ms ease-out;
 }
 
+.settings .settings-tab > * {
```

Comment:
Maybe better to just enclose the tab in the click listener or check up the tree?

Reply:
Ah yeah this was some residual hackathon laziness :laughing: 

Fixed by using a `<Link>` component for `SettingsTab` and getting rid of the manual `onClick` handler